impl/src/main/java/org/glassfish/soteria/identitystores/LdapIdentityStore.java

```diff
@@ -358,7 +373,11 @@ private static LdapContext createLdapContext(String url, String bindDn, String b
         environment.put(SECURITY_PRINCIPAL, bindDn);
         environment.put(SECURITY_CREDENTIALS, bindCredential);
 
-        return environment;
+        try {
```

Comment:
There's really no need to catch `NamingException` and rethrow it, since we already declare that we throw `NamingException`, just let it propagate -- no try/catch needed.